code/modules/events/carp_migration.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/carp_migration
 	name = "Carp Migration"
 	typepath = /datum/round_event/carp_migration
-	weight = 15
+	weight = 10

```

Comment:
So basically you added a 50% chance to make carp non hostile?

Whats the point?